src/main/resources/tanagra/cms_synpuf/expanded/entity/condition.json

```diff
@@ -8,19 +8,19 @@
     "displayHint" : {
       "type" : "ENUM",
       "enumVals" : [ {
-        "enumVal" : {
-          "value" : { },
-          "display" : "Source"
-        },
-        "count" : 0
-      }, {
         "enumVal" : {
           "value" : {
             "stringVal" : "S"
           },
           "display" : "Standard"
         },
-        "count" : 97214
+        "count" : 965
```

Comment:
Why are the counts hardcoded here?

Reply:
The data in `expanded` is generated by querying the source dataset. My thinking for MVP is that our indexer will run both `EXPAND_CONFIG` and `INDEX_ALL` at the same time, and we will no longer need to check in the expanded data at all.

As part of that, it might be helpful to figure out what from `expanded` is needed at index time, and what's needed by the UI code, so the data stored for the UI can be minimal.